### PR TITLE
[CDF-3879] FunctionsAPI.create() handles name with / character

### PR DESCRIPTION
**Pull request by HMEiding**
Ideally we would have a test for this, which would be an integration tests. However, we don't have integration tests for functions yet, primarily because creating functions takes some time.

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -18,7 +18,7 @@ line_length=120                # corresponds to -w  flag
 multi_line_output=3            # corresponds to -m  flag
 include_trailing_comma=true    # corresponds to -tc flag
 skip_glob = '^((?!py$).)*$'    # this makes sort all Python files
-known_third_party = ["black", "pytest", "requests", "responses", "typing-extensions"]
+known_third_party = ["black", "pytest", "requests", "responses", "typing_extensions"]
```
Comment (HMEiding) [inline pyproject.toml:21]: Not sure why, but this was forced by the pre-commit hooks.

Comment (sanderland) [reply pyproject.toml:21]: @pavelzubarev changed this the other way -- let's wait for him on why

Comment (pavelzubarev) [reply pyproject.toml]: that was changed because typing_extension doesn't exist on PyPi, so my dependency check was failing.

Comment (sanderland) [reply pyproject.toml]: but this is an isort parameter, not a dependency

Comment (sanderland) [reply pyproject.toml]: yeah confirmed he search-and-replaced a bit too aggressively :D merge ahead @HMEiding

**Pull request merged**